.github/workflows/lint.yml

```diff
@@ -46,3 +46,6 @@ jobs:
             pre-commit install --hook-type pre-push
       - name: Run linter
         run: make plugin=${{ inputs.plugin }} lint
+      - name: Run mypy
```

Comment:
This step will be removed when we fix issues on `kedro-datasets` - `mypy` check will run as part of the previous step `make plugin=<plugin> lint`